package.xml

```diff
@@ -996,8 +996,8 @@ Update to wrap gRPC C Core version 0.10.0
   </release>
   <release>
    <version>
-    <release>0.14.0</release>
-    <api>0.14.0</api>
+    <release>0.8.1</release>
```

Comment:
So this is a bit backward. This line of code you are looking at is at the bottom of the file, in the area of listing all the history of releases. However, in the past, I had (incorrectly) replaced the release number there, with a templatized release string. Since this is the generated file, it contains whatever it's on the current release, even though the content of that block, did refer to the old 0.8.1 release. 

Anyway, I am pretty sure this is correct. It's just that the release string is in multiple places. This is the history section which adds to the confusion.